docker/qgisserver/tests/functional/accesscontrol_test.py

```diff
@@ -69,6 +69,10 @@ def test_data(clean_dbsession):
 
     dbsession = DBSession()
 
+    for cls in [RestrictionArea, LayerWMS, OGCServer, Role, User]:
+        for obj in dbsession.query(cls).all():
+            dbsession.delete(obj)
```

Comment:
Not sure this is stricly needed

Reply:
No it shouldn't, but I'm more confident with that :-)